diff -up akonadi-contacts-20.03.90/kcm/kcmakonadicontactactions.cpp.omv~ akonadi-contacts-20.03.90/kcm/kcmakonadicontactactions.cpp
--- akonadi-contacts-20.03.90/kcm/kcmakonadicontactactions.cpp.omv~	2020-04-09 19:36:56.552467778 +0200
+++ akonadi-contacts-20.03.90/kcm/kcmakonadicontactactions.cpp	2020-04-09 19:38:26.323609914 +0200
@@ -24,6 +24,8 @@
 
 #include "contactactionssettings.h"
 
+#include <qglobal.h>
+
 #include <KAboutData>
 #include <KConfigDialogManager>
 #include <KPluginFactory>
@@ -54,7 +56,7 @@ KCMAkonadiContactActions::KCMAkonadiCont
     ui.DialPhoneNumberAction->addItem(i18n("Ekiga"), ContactActionsSettings::UseEkiga);
     ui.DialPhoneNumberAction->addItem(i18n("SflPhone"), ContactActionsSettings::UseSflPhone);
     ui.DialPhoneNumberAction->addItem(i18n("External Application"), ContactActionsSettings::UseExternalPhoneApplication);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
     connect(ui.DialPhoneNumberAction, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &KCMAkonadiContactActions::slotDialPhoneNumberActionChanged);
 #else
     connect(ui.DialPhoneNumberAction, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &KCMAkonadiContactActions::slotDialPhoneNumberActionChanged);
@@ -63,7 +65,7 @@ KCMAkonadiContactActions::KCMAkonadiCont
     ui.SendSmsAction->addItem(i18n("Skype"), ContactActionsSettings::UseSkypeSms);
     ui.SendSmsAction->addItem(i18n("SflPhone"), ContactActionsSettings::UseSflPhoneSms);
     ui.SendSmsAction->addItem(i18n("External Application"), ContactActionsSettings::UseExternalSmsApplication);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
     connect(ui.SendSmsAction, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &KCMAkonadiContactActions::slotSmsPhoneNumberActionChanged);
 #else
     connect(ui.SendSmsAction, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &KCMAkonadiContactActions::slotSmsPhoneNumberActionChanged);
@@ -73,7 +75,7 @@ KCMAkonadiContactActions::KCMAkonadiCont
     ui.ShowAddressAction->addItem(i18n("Google map"), ContactActionsSettings::UseGooglemap);
     ui.ShowAddressAction->addItem(i18n("Map quest"), ContactActionsSettings::UseMapquest);
     ui.ShowAddressAction->addItem(i18n("OpenStreetMap"), ContactActionsSettings::UseOpenStreetMap);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
     connect(ui.ShowAddressAction, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &KCMAkonadiContactActions::slotShowAddressActionChanged);
 #else
     connect(ui.ShowAddressAction, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &KCMAkonadiContactActions::slotShowAddressActionChanged);
